packages/eddsa-poseidon/tests/index.test.ts

```diff
@@ -227,6 +238,20 @@ describe("EdDSAPoseidon", () => {
         expect(unpackedPublicKey[1]).toBe(publicKey[1])
     })
 
+    it("Should not unpack a public key if the public key is not on the curve", async () => {
+        const fun = () => unpackPublicKey("e")
+
+        expect(fun).toThrow("Invalid public key")
```

Comment:
Isn't this a type error?

Reply:
@0xjei If it's a type error, the test message is not the right one.

Reply:
What kind of message do you prefer?

Reply:
If it's a type error, I think you should either specify that it's throwing a type error or changing the input to meet the test message.

Reply:
Hmm the current message is "Should not unpack a public key if the public key type is not supported" so I mentioning the type. I think you're seeing an outdated view

Reply:
Yes, I saw the old message! It's ok then 👍🏽